cranelift/isle/isle/src/ast.rs

```diff
@@ -419,3 +420,26 @@ pub enum ArgPolarity {
     /// *from* the extractor op.
     Output,
 }
+
+/// An implicit converter, which can appear in extractor or
+/// constructor position. This sets up an ambient automatic type
+/// conversion that is used when needed.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Converter {
```

Comment:
This also doesn't seem to differentiate between conversions in an extraction vs construction context, which we talked about in the issue.

Reply:
I don't think this is needed here: even if we do move to a terms-as-converters rather than explicit-converter-funcs design, we can look up the term and find an attached extractor or converter, so we don't need to record directionality here, I think.